server/sonar-web/src/main/js/apps/issues/styles.css

```diff
@@ -135,6 +135,7 @@
 
 .issues .issue:hover {
   border-color: #dd4040;
+  border-style: dashed;
```

Comment:
If I hover the selected issue I see the dashed border. But I think the selected style with solid border should be displayed.